tiles_xyz/tiles_xyz_algorithm.py

```diff
@@ -301,7 +322,7 @@ def set_parameters(self, tile_params):
             tile_format = 'JPEG'
             options = ['QUALITY=%s' % tile_params.get('quality', 75)]
         else:
-            options = ['ZLEVEL=8']
+            options = ['ZLEVEL=%s' % math.ceil(tile_params.get('quality', 75)*9/100)]
```

Comment:
I wouldn't interpret 'quality' for PNG as compression level. It is lossless compression, so no change in quality. And I do not really see a point in trying to further compress PNG images - they use better compression already than deflate. How about we do not override ZLEVEL at all and leave it to the default (6) - probably less CPU times wasted trying to compress something that's well compressed already.